FlorenceBML/BMLacq812/BMLacq812.xml

```diff
@@ -37,6 +38,177 @@ schematypens="http://relaxng.org/ns/structure/1.0"
+                    <msContents>
+                        <summary/>
+                        <msItem xml:id="ms_i1">
+                            <locus from="1" to="185"/>
+                            <title ref="LIT2701Dawit" type="complete"/>
+                            <msItem xml:id="ms_i1.1">
+                                <locus from="1" to="140r"/>
+                                <title ref="LIT2000Mazmur" type="incomplete">150 Psalms</title>
+                                <note>Pseudoepigraphical Psalm no. 151 attached at <locus from="139v" to="140r"/></note>
+                                <note>Written with black ink; red ink for incipits and titles.</note>
+                            </msItem>
+                            <msItem xml:id="ms_i1.2">
+                                <locus from="140r" to="158" type="incomplete"/>
+                                <title ref="LIT1828Mahale">14 Songs of the Prophets</title>
+                                <msItem xml:id="ms_i1.2.1">
+                                    <locus from="140r" to="141v"/>
+                                    <title ref="LIT2277salotu"/>
+                                </msItem>
+                                <msItem xml:id="ms_i1.2.2">
+                                    <locus from="141v" to="143r"/>
+                                    <title ref="LIT2278salota"/>
+                                </msItem>
+                                <msItem xml:id="ms_i1.2.3">
+                                    <locus from="143r" to="145r"/>
+                                    <title ref="LIT2703salota"/>
+                                </msItem>
+                                <msItem xml:id="ms_i1.2.4">
+                                    <locus from="145r" to="146r"/>
+                                    <title ref="LIT2257salota"/>
+                                </msItem>
+                                <msItem xml:id="ms_i1.2.5">
+                                    <locus from="146" to="147r"/>
+                                    <title ref="LIT2258salota"/>
+                                </msItem>
+                                <msItem xml:id="ms_i1.2.6">
+                                    <locus from="147" to="148r"/>
+                                    <title ref="LIT2265salota"/>
+                                </msItem>
+                                <msItem xml:id="ms_i1.2.7">
+                                    <locus from="148r" to="148v"/>
+                                    <title ref="LIT2274salota"/>
+                                </msItem>
+                                <msItem xml:id="ms_i1.2.8">
+                                    <locus from="148v" to="150r"/>
+                                    <title ref="LIT2250salota"/>
+                                </msItem>
+                                <msItem xml:id="ms_i1.2.9">
+                                    <locus from="150" to="151r"/>
```

Comment:
150r?

Reply:
I took it, as it is given in Marrassinis catalogue. If the images are available to me, I will update the folio numbering.

Reply:
I saw the images and committed some changes.